File: commonLib/src/main/java/com/travel/commonlib/transfer/transferPaxModels/MobileCountryCode.java

package com.travel.commonlib.transfer.transferPaxModels;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class MobileCountryCode implements Serializable {

    @SerializedName("CountryId")
    @Expose
    private Integer countryId;
    @SerializedName("CountryName")
    @Expose
    private String countryName;
    @SerializedName("CountryNameArabic")
    @Expose
    private String countryNameArabic;
    @SerializedName("CountryCode")
    @Expose
    private String countryCode;
    @SerializedName("MobileCode")
    @Expose
    private String mobileCode;
    @SerializedName("CurrencyCode")
    @Expose
    private String currencyCode;
    @SerializedName("IsdCode")
    @Expose
    private String isdCode;
    @SerializedName("Ranking")
    @Expose
    private Integer ranking;
    @SerializedName("MobileCodeArabic")
    @Expose
    private String mobileCodeArabic;
    @SerializedName("culture")
    @Expose
    private Integer culture;
    @SerializedName("CountryFormat")
    @Expose
    private Integer countryFormat;
    @SerializedName("CountryCodeFormat")
    @Expose
    private Integer countryCodeFormat;
    @SerializedName("CountrynameFormatted")
    @Expose
    private String countrynameFormatted;
    @SerializedName("CountryCodeFormatted")
    @Expose
    private String countryCodeFormatted;
    @SerializedName("CountryNumber")
    @Expose
    private Object countryNumber;

    public Integer getCountryId() {
        return countryId;
    }

    public void setCountryId(Integer countryId) {
        this.countryId = countryId;
    }

    public String getCountryName() {
        return countryName;
    }

    public void setCountryName(String countryName) {
        this.countryName = countryName;
    }

    public String getCountryNameArabic() {
        return countryNameArabic;
    }

    public void setCountryNameArabic(String countryNameArabic) {
        this.countryNameArabic = countryNameArabic;
    }

    public String getCountryCode() {
        return countryCode;
    }

    public void setCountryCode(String countryCode) {
        this.countryCode = countryCode;
    }

    public String getMobileCode() {
        return mobileCode;
    }

    public void setMobileCode(String mobileCode) {
        this.mobileCode = mobileCode;
    }

    public String getCurrencyCode() {
        return currencyCode;
    }

    public void setCurrencyCode(String currencyCode) {
        this.currencyCode = currencyCode;
    }

    public String getIsdCode() {
        return isdCode;
    }

    public void setIsdCode(String isdCode) {
        this.isdCode = isdCode;
    }

    public Integer getRanking() {
        return ranking;
    }

    public void setRanking(Integer ranking) {
        this.ranking = ranking;
    }

    public String getMobileCodeArabic() {
        return mobileCodeArabic;
    }

    public void setMobileCodeArabic(String mobileCodeArabic) {
        this.mobileCodeArabic = mobileCodeArabic;
    }

    public Integer getCulture() {
        return culture;
    }

    public void setCulture(Integer culture) {
        this.culture = culture;
    }

    public Integer getCountryFormat() {
        return countryFormat;
    }

    public void setCountryFormat(Integer countryFormat) {
        this.countryFormat = countryFormat;
    }

    public Integer getCountryCodeFormat() {
        return countryCodeFormat;
    }

    public void setCountryCodeFormat(Integer countryCodeFormat) {
        this.countryCodeFormat = countryCodeFormat;
    }

    public String getCountrynameFormatted() {
        return countrynameFormatted;
    }

    public void setCountrynameFormatted(String countrynameFormatted) {
        this.countrynameFormatted = countrynameFormatted;
    }

    public String getCountryCodeFormatted() {
        return countryCodeFormatted;
    }

    public void setCountryCodeFormatted(String countryCodeFormatted) {
        this.countryCodeFormatted = countryCodeFormatted;
    }

    public Object getCountryNumber() {
        return countryNumber;
    }

    public void setCountryNumber(Object countryNumber) {
        this.countryNumber = countryNumber;
    }

}

File: commonLib/src/main/java/com/travel/commonlib/transfer/TransfersGetPax/MobileCountryCode.kt
package com.travel.commonlib.transfer.TransfersGetPax

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose
import java.io.Serializable

class MobileCountryCode : Serializable {
    @SerializedName("CountryId")
    @Expose
    var countryId: Int? = null

    @SerializedName("CountryName")
    @Expose
    var countryName: String? = null

    @SerializedName("CountryNameArabic")
    @Expose
    var countryNameArabic: String? = null

    @SerializedName("CountryCode")
    @Expose
    var countryCode: String? = null

    @SerializedName("MobileCode")
    @Expose
    var mobileCode: String? = null

    @SerializedName("CurrencyCode")
    @Expose
    var currencyCode: String? = null

    @SerializedName("IsdCode")
    @Expose
    var isdCode: String? = null

    @SerializedName("Ranking")
    @Expose
    var ranking: Int? = null

    @SerializedName("MobileCodeArabic")
    @Expose
    var mobileCodeArabic: String? = null

    @SerializedName("culture")
    @Expose
    var culture: Int? = null

    @SerializedName("CountryFormat")
    @Expose
    var countryFormat: Int? = null

    @SerializedName("CountryCodeFormat")
    @Expose
    var countryCodeFormat: Int? = null

    @SerializedName("CountrynameFormatted")
    @Expose
    var countrynameFormatted: String? = null

    @SerializedName("CountryCodeFormatted")
    @Expose
    var countryCodeFormatted: String? = null

    @SerializedName("CountryNumber")
    @Expose
    var countryNumber: Any? = null

    @SerializedName("CountryUNCode")
    @Expose
    var countryUNCode: Any? = null
}